### PR TITLE
Fixed #223: Bug while listing remote envs

### LINKED ISSUE #223: [Bug]: Error while listing environments from remote
image

### PR DESCRIPTION
**Pull request by asa1997**
resolves #223

### CODE REVIEW

File: src/main/bash/commands/besman-list.sh [hunk 1]
```diff
@@ -117,7 +117,7 @@ function __besman_check_repo_exist()
     repo=$(echo "$BESMAN_ENV_REPO" | cut -d "/" -f 2)
     repo_url="https://api.github.com/repos/$namespace/$repo"
 
-    response=$(curl --head --silent "$repo_url" | head -n 1 | awk '{print $2}')
+    response=$(curl --head --insecure -s --silent "$repo_url" | head -n 1 | awk '{print $2}')
```
Comment (pramit-d) [inline src/main/bash/commands/besman-list.sh:120]: @asa1997 Why are both `-s` and `--silent` required in a same `curl `command?

**Review by pramit-d — CHANGES_REQUESTED** [review]

File: src/main/bash/commands/besman-list.sh (hunk 1 above)
Comment (asa1997) [reply src/main/bash/commands/besman-list.sh]: Removed.

**Review by pramit-d — APPROVED** [review]
@asa1997 LGTM

**Review by harimohanr — APPROVED** [review]
@asa1997 Looks okay

**Pull request merged**